day00/ex10/ex10.md

```diff
@@ -0,0 +1,57 @@
+# Exercise 10 - Vectorized Mean Squared Error as a linear cost function
+
+|                         |                    |
+| -----------------------:| ------------------ |
+|   Turnin directory :    |  ex10              |
+|   Files to turn in :    |  vec_linear_mse.py  |
+|   Forbidden function :  |  None              |
+|   Remarks :             |  n/a               |
+
+**Objective:**
+You must implement the following formula as a function:  
+
+$$
+MSE = \frac{1}{m}\sum_{i=1}^{m}(X\theta - y)^{T}}(X\theta - y)
```

Comment:
X\^{T}\cdot theta
no index i inside the summation

Reply:
Not sure here either : the vectors in X are rows and you do not have to transpose before doing a dot product